boards/phytec/phycore_rt1170/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+if(CONFIG_NXP_IMXRT_BOOT_HEADER)
+  zephyr_library()
+  if(NOT ((DEFINED CONFIG_BOARD_PHYCORE_RT1170_MIMXRT1176_CM7)
+    OR (DEFINED CONFIG_BOARD_PHYCORE_RT1170_MIMXRT1176_CM4)))
+    message(WARNING "It appears you are using the board definition for "
+     "the phyCORE-RT1170, but targeting a custom board. You may need to "
+     "update your flash configuration or device configuration data blocks")
+  endif()
+  set(RT1170_BOARD_NAME "evkmimxrt1170")
+  set(RT1170_BOARD_DIR
+    "${ZEPHYR_HAL_NXP_MODULE_DIR}/mcux/mcux-sdk/boards/${RT1170_BOARD_NAME}")
+  if(CONFIG_BOOT_FLEXSPI_NOR)
+    # Include flash configuration block for RT1170 EVK from NXP's HAL.
+    # This configuration block may need modification if another flash chip is
+    # used on your custom board. See NXP AN12238 for more information.
+    zephyr_compile_definitions(XIP_BOOT_HEADER_ENABLE=1)
+    zephyr_compile_definitions(BOARD_FLASH_SIZE=CONFIG_FLASH_SIZE*1024)
+    zephyr_library_sources(${RT1170_BOARD_DIR}/xip/${RT1170_BOARD_NAME}_flexspi_nor_config.c)
+    zephyr_library_include_directories(${RT1170_BOARD_DIR}/xip)
+  endif()
+  if(CONFIG_DEVICE_CONFIGURATION_DATA)
+    # Include device configuration data block for RT1170 EVK from NXP's HAL.
+    # This configuration block may need modification if another SDRAM chip
+    # is used on your custom board.
+    zephyr_compile_definitions(XIP_BOOT_HEADER_DCD_ENABLE=1)
```

Comment:
Same request as for the flash configuration block- if you aren't using the same SDRAM part as the RT1170 EVK on your custom board, then please define an new DCD block for this board.